_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,324 @@
+          const gamma = !disableMotion.checked ? event.gamma : 0;
+
+          if (gamma > 20) {
+            slider.value++;
+          } else if (gamma < -20) {
+            slider.value--;
+            output.innerHTML = slider.value;
+          }
+          output.innerHTML = slider.value;
+        }
+
+        window.addEventListener('deviceorientation', handleOrientation);
+      }
+    </script>
+  </head>
+
+  <body onload="activateSlider();">
+    <h1>Slider Motion Sensor Example </h1>
+
+    <p>Open this slider on a device with a motion sensor, such as a smart phone or tablet. Tilt the device to the right and left to adjust the slider value. The check box disables the motion sensing adjustment.</p>
+    <p>Note: This example may not work across all browsers.</p>
+
+    <div>
+      <input type="range" min="1" max="100" value="50" id="motionSlider" disabled>
+      <p aria-live="polite">Slider Value: <span id="output">50</span></p>
+    </div>
+    <div style="position: absolute;margin-left: -9999px;">
+      <input type="checkbox" id="disableMotion">
+      <label for="disableMotion">Disable Motion Actuation</label>
+    </div>
+  </body>
+</html>
+```
+
+#### Failed Example 4
+
+The [HTML document][] has [functionality][] that can be operated by [device motion][] and [can be disabled][], and the control for disabling is [visible][] and [included in the accessibility tree][] but with an [accessible name][] that is empty ("").
+
+```html
+<html>
+  <head>
+    <title>Failed Example 4</title>
+    <script>
+      function activateSlider() {
+        const slider = document.getElementById('motionSlider');
+        const output = document.getElementById('output');
+
+        function handleOrientation(event) {
+          const disableMotion = document.getElementById('disableMotion');
+          const gamma = !disableMotion.checked ? event.gamma : 0;
+
+          if (gamma > 20) {
+            slider.value++;
+          } else if (gamma < -20) {
+            slider.value--;
+            output.innerHTML = slider.value;
+          }
+          output.innerHTML = slider.value;
+        }
+
+        window.addEventListener('deviceorientation', handleOrientation);
+      }
+    </script>
+  </head>
+
+  <body onload="activateSlider();">
+    <h1>Slider Motion Sensor Example </h1>
+
+    <p>Open this slider on a device with a motion sensor, such as a smart phone or tablet. Tilt the device to the right and left to adjust the slider value. The check box disables the motion sensing adjustment.</p>
+    <p>Note: This example may not work across all browsers.</p>
+
+    <div>
+      <input type="range" min="1" max="100" value="50" id="motionSlider" disabled>
+      <p aria-live="polite">Slider Value: <span id="output">50</span></p>
+    </div>
+    <div>
+      <input type="checkbox" id="disableMotion">
+      <label for="disableMotion"></label>
+    </div>
+  </body>
+</html>
+```
+
```

Comment:
We need an example where the control for disabling is not a UI component. (that's the last part of the expectation which is currently not tested)